### PR TITLE
Revamp installation a bit

### PR DESCRIPTION
**Pull request by Keats**
Also removes some linebreaks as it screws up glossary rendering

### CODE REVIEW

**Comment by Keats** [comment]
I had no idea what to put in the nix intro in the readme, feel free to change it

**Comment by teh** [comment]
looks good for now

LGTM, thanks!

File: installation/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+The current recommended way to install Nix on all platforms (apart from Archlinux, see below) is to run the following command into a terminal as your normal user (*not* root):
+
+```bash
+bash <(curl https://nixos.org/nix/install)
+```
+
+While this is *incredibly dangerous and generally discouraged*, using a package manager implies trusting the people who package the code so in this specific case it's OK to trust the first step as well.
+
+> **Note for Archlinux users** <br>The script currently does not work on Archlinux due to https://github.com/NixOS/nix/issues/599 but there is the [nix-multiuser](https://aur.archlinux.org/packages/nix-multiuser/) package in AUR that you can install.
```
Comment (teh) [inline installation/README.md:16]: Maybe we should move this into a separate chapter?

**Comment by teh** [comment]
Other than maybe adding an arch section please merge so I can start changing e.g. the glossary in another branch

**Pull request merged**